scrunch/datasets.py

```diff
@@ -1882,3 +1989,56 @@ def move(self, path, position=-1):
             )
         target_group = self.dataset.order[str(path)]
         target_group.insert(self.alias, position=position)
+
+    def summary(self, names=False, tablefmt='psql'):
```

Comment:
originally I was thinking it was a good idea to just pass over the possibility to change the output (i.e. writing some reStructured Text with tablefmt='rst'). Sure, this is an edge case. For displaying purpose I thaught the psql format would be nice because users might know the format from PostgreSQL.
But as @rdrivers mentioned, I could also just make it a pandas Dataframe.